docs/content/v2024.2/yugabyte-platform/manage-deployments/edit-universe.md

```diff
@@ -52,6 +52,20 @@ YugabyteDB automatically ensures that new nodes start hosting the tablet leaders
 
 To change the number of nodes of universes created with an on-premises cloud provider and secured with third-party certificates obtained from external certification authorities, you must first add the certificates to the nodes you will add to the universe. Refer to [Add certificates](../../security/enable-encryption-in-transit/add-certificate-ca/). Ensure that the certificates are signed by the same external CA and have the same root certificate. In addition, ensure that you copy the certificates to the same locations that you originally used when creating the universe.
 
+### Edit connection pooling
+
+{{<tags/feature/tp>}}If your universe is running database v2024.2 or later, you can enable [Built-in connection pooling](../../../explore/going-beyond-sql/connection-mgr-ysql/).
+
+While in Tech Preview, the feature is not available by default. To make connection pooling available, set the **Allow users to enable or disable connection pooling** Global Runtime Configuration option (config key `yb.universe.allow_connection_pooling`) to true. Refer to [Manage runtime configuration settings](../../administer-yugabyte-platform/manage-runtime-config/). You must be a Super Admin to set global runtime configuration flags.
+
+To enable or disable connection pooling on a universe:
```

Comment:
Do we want to show the steps to enable connection pooling on a fresh universe too?